app_dart/test/src/request_handling/fake_http.dart

```diff
@@ -858,6 +858,10 @@ class FakeHttpClientRequest extends FakeOutbound implements HttpClientRequest {
 
   @override
   Future<HttpClientResponse> get done => _doneCompleter.future;
+
+  // @override
+  // ignore: annotate_overrides
+  void abort([Object error, StackTrace stackTrace]) {}
```

Comment:
This is being added to Dart as of https://github.com/dart-lang/sdk/commit/56d40bad75255688772a6b306fd459a59cd737c8

I'm not clear on whether we're testing against flutter stable or master, so for now this makes it compatble with both. We can uncomment the annotation and remove the ignore once the change rolls to stable (or cocoon is using master).

Reply:
For cocoon we are using beta channel.